.expeditor/config.yml

```diff
@@ -14,34 +14,38 @@ github:
   delete_branch_on_merge: true
   # The tag format to use (e.g. v1.0.0)
   version_tag_format: "win32-taskscheduler-{{version}}"
-  release_branch:
-    - master:
-        version_constraint: "0.*"
-    - ole:
-        version_constraint: "2.*"
-promote:
-  action:
-    - built_in:rollover_changelog
-    - built_in:publish_rubygems
 
-  # allow bumping the minor release via label
-  minor_bump_labels:
-    - "Expeditor: Bump Version Minor"
+release_branches:
+  - master:
+      version_constraint: "0.*"
+  - ole:
+      version_constraint: "2.*"
 
 changelog:
   rollup_header: Changes not yet released to rubygems.org
 
-# These actions are taken, in order they are specified, anytime a Pull Request is merged.
-merge_actions:
-  - built_in:bump_version:
-      ignore_labels:
-        - "Expeditor: Skip Version Bump"
-        - "Expeditor: Skip All"
-  - bash:.expeditor/update_version.sh:
-      only_if: built_in:bump_version
-  - built_in:update_changelog:
-      ignore_labels:
-        - "Expeditor: Skip Changelog"
-        - "Expeditor: Skip All"
-  - built_in:build_gem:
-      only_if: built_in:bump_version
+subscriptions:
+  # These actions are taken, in order they are specified, anytime a Pull Request is merged.
+  - workload: pull_request_merged:{{github_repo}}:{{release_branch}}:*
+    actions:
+      - built_in:bump_version:
+          ignore_labels:
+            - "Expeditor: Skip Version Bump"
+            - "Expeditor: Skip All"
+      - bash:.expeditor/update_version.sh:
+          only_if: built_in:bump_version
+      - built_in:update_changelog:
+          ignore_labels:
+            - "Expeditor: Skip Changelog"
+            - "Expeditor: Skip All"
+      - built_in:build_gem:
+          only_if: built_in:bump_version
+
+  - workload: project_promoted:{{agent_id}}:*
+    actions:
+      - built_in:rollover_changelog
+      - built_in:publish_rubygems
+      
+    # allow bumping the minor release via label
```

Comment:
This should be nested under `github` right below (and at the same indentation as) `version_tag_format`